packages/@aws-cdk/aws-logs/lib/log-group.ts

```diff
@@ -273,6 +274,13 @@ export enum RetentionDays {
  * Properties for a LogGroup
  */
 export interface LogGroupProps {
+  /**
+   * The KMS Key to encrypt the log group with.
+   *
+   * @default No KMS key
+   */
+  readonly kmsKey?: IKey;
```

Comment:
we commonly call this `encryptionKey` unless there's a need for multiple keys in which case we tend to prefix it with the qualifier that the key applies to.
```suggestion
  readonly encryptionKey?: IKey;
```